### PR TITLE
[ko] Set() 생성자 신규 번역, Set 설명 추가

### PR DESCRIPTION
**Pull request by wisedog**
- Set() 생성자 신규 번역
- 기존 yari 렌더링 문제로 추가하지 못했던 `Set composition`이 추가됨

Geek News에서 [JavaScript 의 새로운 Set 메소드들](https://news.hada.io/topic?id=15586)이 소개되면서 Set 신규 추가를 홍보하기에 노 저어봅니다.

### CODE REVIEW

**Comment by hochan222** [comment]
@caugner Hello!
It seems to be exposed well in the production environment(may be..?), but I don't know what the problem is in the preview link, but the image is not displayed. It would be nice if it could be improved in the future. (or bug..?)

- https://pr22061.content.dev.mdn.mozit.cloud/ko/docs/Web/JavaScript/Reference/Global_Objects/Set

image

File: files/ko/web/javascript/reference/global_objects/set/set/index.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+new Set(iterable)
+```
+
+> **참고:** `Set()`은 오직 [`new`](/ko/docs/Web/JavaScript/Reference/Operators/new)와 함께 생성할 수 있습니다. `new` 없이 생성을 시도하면 {{jsxref("TypeError")}}가 발생합니다.
+
+### 매개변수
+
+- `iterable` {{optional_inline}}
+
+  - : [순회 객체](/ko/docs/Web/JavaScript/Reference/Statements/for...of)가 전달되면 모든 요소가 새로운 `Set`에 추가됩니다.
```
Comment (hochan222) [inline files/ko/web/javascript/reference/global_objects/set/set/index.md:27]:
```suggestion
  - : [순회 가능한 객체](/ko/docs/Web/JavaScript/Reference/Statements/for...of)가 전달되면 모든 요소가 새로운 `Set`에 추가됩니다.
```

> iterable object

**Review by hochan222 — CHANGES_REQUESTED** [review]
고생 많으셨습니다~! 아래 한가지 리뷰사항 제안 드립니다.

좋은 하루 보내세요 :)

**Review by hochan222 — APPROVED** [review]
LGTM :)

**Comment by caugner** [comment]
> @caugner Hello!
> It seems to be exposed well in the production environment(may be..?), but I don't know what the problem is in the preview link, but the image is not displayed. It would be nice if it could be improved in the future. (or bug..?)

Unfortunately, it is a known issue that the preview does not render images that belong to a different page that hasn't been modified in the same PR.

@yin1999 I believe you noticed this before. Do you remember if you opened an issue in yari?

**Comment by yin1999** [comment]
> > @caugner Hello!
> > It seems to be exposed well in the production environment(may be..?), but I don't know what the problem is in the preview link, but the image is not displayed. It would be nice if it could be improved in the future. (or bug..?)
> 
> Unfortunately, it is a known issue that the preview does not render images that belong to a different page that hasn't been modified in the same PR.
> 
> @yin1999 I believe you noticed this before. Do you remember if you opened an issue in yari?

The previously opened issue does not seem to be this. I believe the issue in the current PR  is related to this line:

https://github.com/mdn/yari/blob/58353d37287a3e9d596a4d002eb4c08e5ae8cae3/build/check-images.ts#L156

This line only considers the case where the image file is directly in the folder where the current document is located.

Actually, we should get the relative path instead of the `basename` (filename only) for the images in other paths (that is, not only the locale is different. e.g. `/ko/docs/Web/JavaScript/Reference/Global_Objects/Set/difference` againsts `/ko/docs/Web/JavaScript/Reference/Global_Objects/Set`).

**Comment by hochan222** [comment]
@yin1999 , Um... I see. Thank you. 

I won't create a issue on yari, but I'll note it.

**Pull request merged**